optimum/intel/openvino/modeling_diffusion.py

```diff
@@ -116,15 +118,23 @@ def __init__(
         else:
             self._model_save_dir = model_save_dir
 
-        self.vae_decoder = OVModelVaeDecoder(vae_decoder, self)
+        default_vae_ov_config = deepcopy(self.ov_config)
+        if "GPU" in self._device:
+            default_vae_ov_config.update({"INFERENCE_PRECISION_HINT": "f32"})
```

Comment:
I guess it can be done simplier, we can just check that inference precision hint set in config before and do not update it if it was provided

Reply:
Thanks for the suggestion! Updated